Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ui/settings/notification/NotificationsSettingsTest.kt

```diff
@@ -25,27 +25,27 @@ class NotificationsSettingsTest : BaseTest() {
 
     @MockK lateinit var foregroundState: ForegroundState
     @MockK lateinit var notificationManagerCompat: NotificationManagerCompat
+    @MockK lateinit var cwaSettings: CWASettings
 
     @BeforeEach
     fun setup() {
         MockKAnnotations.init(this)
         mockkObject(LocalData)
 
-        every { LocalData.isNotificationsRiskEnabledFlow } returns flow { emit(true) }
-        every { LocalData.isNotificationsRiskEnabled = any() } just Runs
-        every { LocalData.isNotificationsRiskEnabled } returns true
-
-        every { LocalData.isNotificationsTestEnabledFlow } returns flow { emit(true) }
-        every { LocalData.isNotificationsTestEnabled = any() } just Runs
-        every { LocalData.isNotificationsTestEnabled } returns true
+        every { cwaSettings.isNotificationsRiskEnabledFlow.flow } returns flow { emit(true) }
```

Comment:
you could also use
```suggestion
        every { cwaSettings.isNotificationsRiskEnabledFlow } returns mockFlowPreference(true)
```
note that this needs to be imported and im not 100% sure of the function name

Reply:
didn't notice we have this `mockFlowPreference()`, updated